README.md

```diff
@@ -129,15 +130,49 @@ cargo fmt --all
 cargo clippy --workspace --all-features -- -D warnings
 ```
 
+## Release Process
+
+### Overview
+
+We make releases within the `rust` repository specific to the crate(s) that will be consumed (e.g. agents@1.0.0, configuration@1.0.0, accumulator@1.0.0, etc).
```

Comment:
can we make an exhaustive list? so we know exactly what the release words should be?

Reply:
Made list exhaustive both obv subject to change in future. What's on list is what we anticipate releasing today